<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="messageBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sizeComboBox">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>270</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>12</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="boldToolBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/bold.png</normaloff>:/images/images/bold.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="italicToolBtn">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>倾斜</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/italic.png</normaloff>:/images/images/italic.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="underlineToolBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/under.png</normaloff>:/images/images/under.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="colorToolBtn">
   <property name="geometry">
    <rect>
     <x>326</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>更改字体颜色</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/color.png</normaloff>:/images/images/color.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="sendToolBtn">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>传输文件</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/send.png</normaloff>:/images/images/send.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="saveToolBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>保存聊天记录</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="clearToolBtn">
   <property name="geometry">
    <rect>
     <x>439</x>
     <y>270</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>清空聊天记录</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/clear.png</normaloff>:/images/images/clear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>311</width>
     <height>431</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>主机名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>717</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNumLabel">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>453</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>在线用户：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
